code/modules/projectiles/guns/projectile/rocketlauncher.dm

```diff
@@ -6,7 +6,7 @@
 	item_state = "rpg"
 	max_shells = 1
 	w_class = W_CLASS_LARGE
-	starting_materials = list(MAT_IRON = 5000)
+	starting_materials = list(MAT_IRON = 25000, MAT_GLASS = 7500, MAT_PLASTIC = 12500, MAT_GOLD = 3000)
```

Comment:
what the fuck?

Reply:
yes, those are the actual material costs